src/ontology/uberon-edit.obo

```diff
@@ -224271,6 +224271,1025 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+
+[Term]
+id: UBERON:8310096
+name: gray communicating ramus of fourth lumbar nerve
+def: "The postganglionic nerve fibers from the fourth lumbar sympathetic chain ganglion passing to the fourth lumbar spinal nerve." [http://scicrunch.org/resolver/SCR_018709, https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/gray-ramus-communicans]
+synonym: "L4 gray ramus" EXACT []
+synonym: "L4 grey ramus" EXACT []
+xref: FMA:65913
+xref: ILX:0788536
+is_a: UBERON:0017642
+
+[Term]
+id: UBERON:8310097
+name: gray communicating ramus of fifth lumbar nerve
+def: "The postganglionic nerve fibers from the fifth lumbar sympathetic chain ganglion passing to the fifth lumbar spinal nerve." [http://scicrunch.org/resolver/SCR_018709, https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/gray-ramus-communicans]
+synonym: "L5 gray ramus" EXACT []
+synonym: "L5 grey ramus" EXACT []
+xref: FMA:65914
+xref: ILX:0785119
+is_a: UBERON:0017642
+
+[Term]
+id: UBERON:8310098
+name: gray communicating ramus of sixth lumbar nerve
+def: "The postganglionic nerve fibers from the sixth lumbar sympathetic chain ganglion passing to the sixth lumbar spinal nerve. Not present in human" [http://scicrunch.org/resolver/SCR_018709, https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/gray-ramus-communicans]
+synonym: "L6 gray ramus" EXACT []
+synonym: "L6 grey ramus" EXACT []
+xref: ILX:0739299
+is_a: UBERON:0017642
+
+[Term]
+id: UBERON:8310099
+name: gray communicating ramus of seventh lumbar nerve
+def: "The postganglionic nerve fibers from the seventh lumbar sympathetic chain ganglion  passing to the seventh lumbar spinal nerve. Not present in human.  Present in pig, monkey, and cat." [http://scicrunch.org/resolver/SCR_018709, https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/gray-ramus-communicans]
+synonym: "L7 gray ramus" EXACT []
+synonym: "L7 grey ramus" EXACT []
+xref: ILX:0793858
+is_a: UBERON:0017642
+
+[Term]
+id: UBERON:8310100
+name: gray communicating ramus of first sacral nerve
+def: "The postganglionic nerve fibers from the first sacral sympathetic chain ganglion passing to the first sacral spinal nerve." [http://scicrunch.org/resolver/SCR_018709, https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/gray-ramus-communicans]
+synonym: "S1 gray ramus" EXACT []
+synonym: "S1 grey ramus" EXACT []
+xref: FMA:65986
+xref: ILX:0793228
+is_a: UBERON:0017642
+
+[Term]
+id: UBERON:8310101
+name: gray communicating ramus of second sacral nerve
+def: "The postganglionic nerve fibers from the second sacral sympathetic chain ganglion passing to the second sacral spinal nerve." [http://scicrunch.org/resolver/SCR_018709, https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/gray-ramus-communicans]
+synonym: "S2 gray ramus" EXACT []
+synonym: "S2 grey ramus" EXACT []
+xref: FMA:65987
+xref: ILX:0793229
+is_a: UBERON:0017642
```

Comment:
looks like the is_a is too granular here, and need partonomy or similar for S2 (etc.) @stappan

Reply:
actually can't do partonomy to S2 because there is no existing relation that matches it seems